src/lib/utils.ts

```diff
@@ -1,3 +1,5 @@
-export function isEmptyText(text: string | null | undefined): boolean {
+export function isEmptyText<T extends string | null | undefined>(
```

Comment:
Second thought: what's the point of having a separate `utils` namespace in the lib folder. It contains only one function, and I don't see much semantic difference between utils file functions and other functions in the `lib` folder.